src/features/world/ui/megastore/components/ItemsList.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { getItemBuffLabel, getItemImage } from "../MegaStore";
+import { Label } from "components/ui/Label";
+import { pixelDarkBorderStyle } from "features/game/lib/style";
+import { SquareIcon } from "components/ui/SquareIcon";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { formatNumber } from "lib/utils/formatNumber";
+import { getSeasonalTicket } from "features/game/types/seasons";
+
+import token from "assets/icons/token_2.png";
+import { ITEM_DETAILS } from "features/game/types/images";
+import {
+  CollectiblesItem,
+  Currency,
+  InventoryItemName,
+  WearablesItem,
+} from "features/game/types/game";
+
+interface Props {
+  itemsLabel: string;
+  items: (WearablesItem | CollectiblesItem)[];
+  onItemClick: (item: WearablesItem | CollectiblesItem) => void;
+}
+
+export const ItemsList: React.FC<Props> = ({
+  items,
+  itemsLabel,
+  onItemClick,
+}) => {
+  const getCurrencyIcon = (currency: Currency) => {
+    if (currency === "SFL") return token;
+
+    const currencyItem =
+      currency === "Seasonal Ticket" ? getSeasonalTicket() : currency;
+
+    return ITEM_DETAILS[currencyItem as InventoryItemName].image;
+  };
+
+  return (
+    <div className="flex flex-col space-y-2">
+      <div className="bg-brown-300 sticky -top-2 pb-1 z-10">
+        <Label type="info">{itemsLabel}</Label>
```

Comment:
```suggestion
        <Label type="default">{itemsLabel}</Label>
```

The blue label is typically reserved for time boosts or time specific labels.